learning_journal/templates/new_post_template.jinja2

```diff
@@ -117,10 +117,10 @@
     <!-- /.container -->
 
     <!-- jQuery -->
-    <script src="js/jquery.js"></script>
+    <script src="../static/js/jquery.js"></script>
```

Comment:
Consider adding this into your base template instead of importing it into every template.